ambari-server/src/main/java/org/apache/ambari/server/state/SingleSignOnInfo.java

```diff
@@ -0,0 +1,132 @@
+ *   &lt;/sso&gt;
+ * </pre>
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+public class SingleSignOnInfo {
+
+  /**
+   * Indicates whether the relevant service supports SSO integration (<code>true</code>) or not (<code>false</code>).
+   */
+  @XmlElement(name = "supported")
+  private Boolean supported = Boolean.FALSE;
+
+  /**
+   * The configuration that can be used to determine if SSO integration has been enabled.
+   * <p>
+   * It is expected that this value is in the form of <code>configuration-type/property_name</code>
+   */
+  @XmlElement(name = "enabledConfiguration")
+  private String enabledConfiguration = null;
+
+  /**
+   * Default constructor
+   */
+  public SingleSignOnInfo() {
+  }
+
+  /**
+   * Constructor taking in values for supported and the configuration that can be used to determine
+   * if it is enabled for not.
+   *
+   * @param supported            true if SSO integration is supported; false otherwise
+   * @param enabledConfiguration the configuration that can be used to determine if SSO integration has been enabled
+   */
+  public SingleSignOnInfo(Boolean supported, String enabledConfiguration) {
+    this.supported = supported;
+    this.enabledConfiguration = enabledConfiguration;
+  }
+
+  /**
+   * Tests whether the service supports SSO integration.
+   *
+   * @return true if SSO integration is supported; false otherwise
+   */
+  public boolean isSupported() {
+    return Boolean.TRUE.equals(supported);
+  }
+
+  /**
+   * Gets the value whether if the service supports SSO integration.
+   * <p>
+   * <code>null</code> indicates the value was not set
+   *
+   * @return true if SSO integration is supported; false if SSO integration is not supported; null if not specified
+   */
+  public Boolean getSupported() {
+    return supported;
+  }
+
+  /**
+   * Gets the value indicating whether the service supports SSO integration.
+   *
+   * @param supported true if SSO integration is supported; false if SSO integration is not supported; null if not specified
+   */
+  public void setSupported(Boolean supported) {
+    this.supported = supported;
+  }
+
+  /**
+   * Gets the configuration specification that can be used to determine if SSO has been enabled or not.
+   *
+   * @return a configuration specification (config-type/property_name)
+   */
+  public String getEnabledConfiguration() {
+    return enabledConfiguration;
+  }
+
+  /**
+   * Sets the configuration specification that can be used to determine if SSO has been enabled or not.
+   *
+   * @param enabledConfiguration a configuration specification (config-type/property_name)
+   */
+  public void setEnabledConfiguration(String enabledConfiguration) {
+    this.enabledConfiguration = enabledConfiguration;
+  }
+
+  /**
+   * String representation of this object
+   *
+   * @return
+   */
+  @Override
+  public String toString() {
+
+    return "SingleSignOnInfo{" +
```

Comment:
Can you use Objects.toStringHelper() here?

Reply:
Yes.. I was trying to remember what that helper method was and failed to do so.  So I went with the other _standard_ that is typically used.  Thanks.

Reply:
BTW, we need to use `MoreObjects.toStringHelper` now since `Objects.toStringHelper` is deprecated and was supposed to be removed "in June 2016." - https://google.github.io/guava/releases/19.0/api/docs/com/google/common/base/Objects.ToStringHelper.html